topology/doc/Topology_UserManual.tex

```diff
@@ -1483,12 +1462,14 @@ \chapter{Changes from Topology 1.9 to 2.0}\label{sec:changes2}
   Sec.~\ref{sec:conn_masks} and \ref{sec:conn_kernels}.
 \item Functions computing connection probabilities, weights and delays
   as functions of distance between source and target nodes now handle
-  periodic boudary conditions correctly.
+  periodic boundary conditions correctly.
 \item Masks with a diameter larger than the diameter of the layer they
   are applied to are now prohibited by default. This avoids multiple
   connections when masks overwrap.
 \end{itemize}
 
+
+
 \bibliographystyle{plainnat}
 \bibliography{../../extras/bibliography/nest.bib}
 %  \addcontentsline{toc}{chapter}{Bibliography}
```

Comment:
I also have some typos:
- Line 244: 'and' -> 'an' in "...by adding and \lstinline!'extent'!..."
- Line 328: 'place' -> 'placed' in "Elements may be place on the perimeter..."
- Lines 534-535: 'node' -> 'nodes' or 'are' -> 'is' in "... from which source node
  are chosen."
- Line 539: 'source' -> 'target' in "..from which source node
  are chosen."
- Lines 539-540: 'node' -> 'nodes' or 'are' -> 'is' in "...from which source node
  are chosen."
- Line 604: 'I' -> 'In' in "I many cases, the connection dictionary..."
- Line 610: Remove 'the' in "...given, the all neurons in .."
- Line 611: 'consider' -> 'considered' in "...are consider sources or targets."
- Line 799: 'connections' -> 'connection' in "...generates a connections according to this probability."
- Line 818: 'function' -> 'functions' in "More kernel function may be created in a.."
- Line 916: space before 'for' or else we get 'delaysfor' in pdf
- Line 932: drop 'since' or 'as' in "...$(25,0)$ since as the mask extends only..."

I would also change the first 'orange to 'red' and the second 'orange' to 'yellow' in line 620. The color of the circle around the node at (4,5) looks identical to the color around the node at (0,0), which is described as 'light red', and none of the target dots are orange in my eyes. So, "...for the node at $(4,5)$ (marked light orange) are marked with orange dots." -> "...for the node at $(4,5)$ (marked light red) are marked with yellow dots."